src/js/defaults.js

```diff
@@ -0,0 +1,18 @@
+import {validation as v} from './utils';
+
+let chartConfiguration = [{"label": "Value", "formatting": ",.3d"}];
+
+function checkAny(x, defaultVal) {
```

Comment:
`checkAny` is not a good name. it does not tell me that it will return the `defaultVal` if the check (what check) succeeds. 
better would be something like `defaultIfMissingData` or something like that.

Reply:
agreed that it isn't a good name - but the idea is that we may have different types of checks - i.e. I might only care if I have undefined or null - in that case maybe checkUndefinedOrNull (or some better name). checkAny was basically checking any unusual values

Reply:
right, but it's not only checking, right? and the actual check is for missing data. :)